components/Participant.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { createRef, useEffect, useState } from 'react';
+
+const Participant = ({ gain, participant, isYou }) => {
+  const [videoRef] = useState(createRef<HTMLVideoElement>());
+
+  const playTrack = track => {
+    if (track.kind === 'audio') {
+      if (!isYou) {
+        // Don't play your own audio
+        var audio = new Audio();
+        audio.srcObject = postProcessAudioTrack(track);
+      }
+    } else if (track.kind === 'video') {
+      taggedLog('Creating new MediaStream');
+      taggedLog('track:', track);
+      const mediaStreamTrack = track.mediaStreamTrack;
+      const stream = new MediaStream();
+      stream.addTrack(mediaStreamTrack);
+      // @ts-ignore
+      videoRef.current.srcObject = stream;
```

Comment:
You might try a [non-null assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) here!
```suggestion
      videoRef.current!.srcObject = stream;
```

TLDR: It's a way to tell TypeScript "shut up, I know this isn't `undefined` || `null`"